lua/loc/fr.json

```diff
@@ -19,51 +21,72 @@
   "bm_w_model70"        : "Winchester Modèle 70",
 
 	"bm_w_chinchilla"   : "Smith & Wesson Modèle 29",
-  
+
   "bm_w_baka"     : "Micro Uzi IMI",
 
   "bm_w_serbu"   : "Remington 870 à canon court",
-  
+
 	"bm_w_hunter" : "Amur Falcon Pistol Arbalète",
-	"bm_w_china"  : "China Lake Launcher",
+	"bm_w_china"  : "Lanceur China Lake",
 	"bm_w_ray"    : "M202 FLASH Lance-roquettes",
-    
+
 	"bm_wp_upg_o_eotech"        : "EOTech Modèle 553",
   "bm_wp_upg_o_t1micro"       : "Aimpoint  Micro T-1 monté sur rail",
 	"bm_wp_upg_o_docter"        : "Viseur DOCTER II Plus",
+	"bm_wp_upg_o_aimpoint"      : "Optique pour fusils Aimpoint PRO 12841",
 	"bm_wp_upg_o_eotech_xps"    : "EOTech Modèle EXPS3",
 	"bm_wp_upg_o_reflex"        : "Viseur Générique Open Reflex",
-	"bm_wp_upg_o_rx01"          : "Viseur Trijicon RX01 Polarized Reflex",
+	"bm_wp_upg_o_rx01"          : "Viseur Trijicon RX01 Reflex polarisé",
```

Comment:
I've took the liberty to take out the "Polirized" word of this type of Reflex, A reflex is still a reflex anyway.